ohmec_data_ma.js

```diff
@@ -32,51 +32,1926 @@ dataNA = {
       "match":{ "geometryType": "Point"},
       "style":{ "fontname": "Akaya Telivigala"}},
     { "type": "match",
-      "match":{ "entity1name": "Indigenous", "entity2type": "tribe"},
-      "style":{ "strokeOn": false, "fillOpacity": 0.0, "fontname": "New Tegomin", "fontcolor": "#305000", "hifontcolor": "#803000", "borderless": true}},
+      "match":{ "entity1type": "culture"},
+      "style":{ "fontname": "New Tegomin", "fillOpacity": 0.5, "strokeOpacity": 1, "strokeWeight": 2}},
     { "type": "match",
-      "match":{ "entity1type": "nation", "entity1name": "Aztec Alliance"},
-      "style":{ "fontname": "New Tegomin", "fontcolor": "#301000", "strokeColor": "#401008", "fillColor": "#852010", "fillOpacity": 0.5 }}],
+      "match":{ "entity1type": "subculture"},
+      "style":{ "fontname": "New Tegomin", "borderless": true, "hifontcolor": "#59BEDF", "strokeOpacity": 0.0, "fillOpacity": 0.0}}],
```

Comment:
Just as a reminder, `hifontcolor` is the color of the font when highlighted by mousing over the name. This line doesn't have `fontcolor` so it inherits the default. You might want to set both. See the borderless indigenous example in `ohmec_data_na.js`.